app/assets/javascripts/index.js.erb

```diff
@@ -0,0 +1,666 @@
+// ESLINT:
+/* global L, moment, $ */
+

```

Comment:
I think spaces would be better but check what the rest of the codebase uses / confirm with team.